components/claim/claim_controller.jsx

```diff
@@ -7,25 +7,17 @@ import {Route, Switch} from 'react-router-dom';
 
 import logoImage from 'images/logo.png';
 import BackButton from 'components/common/back_button.jsx';
-import OauthToEmail from 'components/claim/components/oauth_to_email';
-import EmailToOauth from 'components/claim/components/email_to_oauth';
-import LdapToEmail from 'components/claim/components/ldap_to_email';
-import EmailToLdap from 'components/claim/components/email_to_ldap';
+import OAuthToEmail from 'components/claim/components/oauth_to_email';
```

Comment:
Converging on a slightly different case here for consistency with the component itself.